tests/testsuite/profiles.rs

```diff
@@ -656,7 +658,32 @@ fn strip_debuginfo_in_release() {
         .run();
 }
 
+// Using -Cstrip=debuginfo in release mode by default is temporarily disabled on Windows due to
+// https://github.com/rust-lang/rust/issues/122857
 #[cargo_test]
+#[cfg(windows)]
```

Comment:
Is this test expected to run also on `windows-gnu`?

```suggestion
#[cfg(all(target_os = "windows", target_env = "msvc"))]
```

Reply:
Thank you :)